projects/adrv9371x/kcu105/system_bd.tcl

```diff
@@ -1,9 +1,12 @@
 
+## FIFO depth is 8Mb - 500kSPS
```

Comment:
I think it is just 'k Samples' and not 'per second'.

Reply:
Yes, you're right, all kSPS or MSPS should be kSamples or MSamples. Thanks!